.github/workflows/CompatHelper.yml

```diff
@@ -0,0 +1,16 @@
+name: CompatHelper
+
+on:
+  schedule:
+    - cron: '00 00 * * *'
+
+jobs:
+  CompatHelper:
+    runs-on: ubuntu-latest
```

Comment:
an array as well?

Reply:
This file was copied from CompatHelper.jl -- it is *exactly* what they suggest one adds. Of course you can change it, but what's the point?

Reply:
The array was needed in a matrix specification and is not needed here.